db/migrate/20180702104632_create_notifications.rb

```diff
@@ -0,0 +1,14 @@
+class CreateNotifications < ActiveRecord::Migration
```

Comment:
Migrating gives an error:
```
Directly inheriting from ActiveRecord::Migration is not supported. Please specify the Rails release the migration was written for:
  class CreateNotifications < ActiveRecord::Migration[4.2]
```
This is due to the Rails 5 update. You can add the text needed by hand, or migrate down this migration, and create a new migration.